### PR TITLE
[ADD] mis_builder: hooks for analytic filtering

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

File: mis_builder/models/aep.py [hunk 1]
```diff
@@ -328,7 +328,7 @@ def get_aml_domain_for_dates(self, date_from, date_to,
         return expression.normalize_domain(domain)
 
     def do_queries(self, date_from, date_to, period_from, period_to,
-                   target_move):
+                   target_move, additional_move_line_filter=[]):
```
Comment (pedrobaeza) [inline mis_builder/models/aep.py]: Same comment as the other PR

**Comment by adrienpeiffer** [comment]
Nice improvement. Thanks ! :+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by fclementic2c** [comment]
Excellent :+1:

**Comment by JordiBForgeFlow** [comment]
:+1 Excellent!

**Pull request merged**